src/GitHub/Data/Webhooks/Payload.hs

```diff
@@ -69,6 +72,7 @@ import           Data.Text                (Text)
 import qualified Data.Text                as T
 import           Data.Vector              (Vector)
 import           GHC.Generics             (Generic)
+import Data.Vector.Fusion.Util (Id)
```

Comment:
What is this used for?

Reply:
Oh oops. The PR is very WIP, I'll let you know once it is ready for review. Thanks :)